flexinput/src/main/java/com/lytefast/flexinput/fragment/PhotosFragment.kt

```diff
@@ -49,7 +50,13 @@ open class PhotosFragment : PermissionsFragment() {
     return view?.apply {
       recyclerView = findViewById(R.id.list)
 
-      val photoAdapter = PhotoCursorAdapter(context.contentResolver, selectionCoordinator!!)
+      val display = activity?.windowManager?.defaultDisplay
+      val size = Point()
+      display?.getSize(size)
+
+      val thumbnailWidth = (size.x / 3f).toInt()
```

Comment:
why are we dividing by 3 here? shouldn't we be doing something based on screen pixel density?

Reply:
The gridview is hardcoded to have 3 columns, ill make this more clear